apps/etterlatte-saksbehandling-ui/client/src/components/person/oppgavebehandling/Oppgavebehandling.tsx

```diff
@@ -0,0 +1,59 @@
+import { useApiCall } from '~shared/hooks/useApiCall'
+import { useEffect } from 'react'
+import { Navigate, Route, Routes } from 'react-router-dom'
+import { getPerson } from '~shared/api/grunnlag'
+import { GYLDIG_FNR } from '~utils/fnr'
+import { StatusBar } from '~shared/statusbar/Statusbar'
+import NavigerTilbakeMeny from '~components/person/NavigerTilbakeMeny'
+import { useNyBehandling } from '~components/person/oppgavebehandling/useNyBehandling'
+import VelgJournalpost from '~components/person/oppgavebehandling/VelgJournalpost'
+import KontrollerOppgave from '~components/person/oppgavebehandling/kontroll/KontrollerOppgave'
+import OpprettNyBehandling from '~components/person/oppgavebehandling/nybehandling/OpprettNyBehandling'
+import { Column, Container, GridContainer } from '~shared/styled'
+import OppsummeringOppgavebehandling from '~components/person/oppgavebehandling/oppsummering/OppsummeringOppgavebehandling'
+import OppgaveStegmeny from '~components/person/oppgavebehandling/OppgaveStegmeny'
+
+export const FEATURE_TOGGLE_KAN_BRUKE_OPPGAVEBEHANDLING = 'pensjon-etterlatte.kan-bruke-oppgavebehandling'
+
+export default function Oppgavebehandling() {
```

Comment:
Er dette oppgavebehandling eller det en spesiell type oppgavebehandling ala opprett behandling for journalpost? 🤠

Reply:
Døper om.